### PR TITLE
Add GameDTO, PlayerDTO, Response

### PR DESCRIPTION
**Pull request by AlekseyPanas**

### CODE REVIEW

**Review by cotopaxi64 — COMMENTED** [review]
GameDTO looks good. However, I don't really get the purpose of the message variable in Response. Is it supposed to be customizable depending on which word was invalid for example? Because if there is only one possible message per ResCode, I don't exactly see the point of messages, because us humans won't be reading the messages in the GUI, and for testing purposes we can just use getCode().

**Comment by AlekseyPanas** [comment]
> GameDTO looks good. However, I don't really get the purpose of the message variable in Response. Is it supposed to be customizable depending on which word was invalid for example? Because if there is only one possible message per ResCode, I don't exactly see the point of messages, because us humans won't be reading the messages in the GUI, and for testing purposes we can just use getCode().

In my opinion, I think having this parameter may be useful. Consider how exceptions in java always come with a message option. A general exception such as IOException might be unclear, but with a message it would clarify the context. Similarly, PlayerNotFound can be thrown in numerous cases, and having a message to clarify what happened can be helpful. Furthermore, we may have errors down the line which might be useful to display as a popup to the user.

**Review by bac423 — APPROVED** [review]
[Documentation Review] Amazing job

**Review by cotopaxi64 — DISMISSED** [review]
Looks good

**Review by Serdiuk-Andrii — DISMISSED** [review]

**Review by bac423 — APPROVED** [review]
[Documentation Review] All good

**Review by Serdiuk-Andrii — DISMISSED** [review]

File: src/main/java/usecases/GameDTO.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+    /**
+     * @return Id of player whose turn it is in this game state data
+     */
+    public String getCurrentTurnPlayerId() { return currentTurnPlayerId; }
+
+    /**
+     * @return List of players who were in the game in this game state data
+     */
+    public ArrayList<PlayerDTO> getPlayers() { return players; }
```
Comment (Serdiuk-Andrii) [inline src/main/java/usecases/GameDTO.java:60]: This can return just a list, but it is not a big deal

**Review by Serdiuk-Andrii — APPROVED** [review]

**Review by bac423 — APPROVED** [review]
[Documentation Review] All good

**Review by cotopaxi64 — APPROVED** [review]
Looks Good

**Review by Serdiuk-Andrii — APPROVED** [review]

**Review by tostistos — APPROVED** [review]

**Pull request merged**